site/docs/patterns/index.mdx

```diff
@@ -0,0 +1,524 @@
+---
+title: Forms
+layout: DetailTechnical
+sidebar:
+  label: Forms
+label: Forms
+---
+
+Forms allow you to capture data, gather opinions and receive information in a consistent format.
+
+This pattern provides practical and visually enriched guidelines for creating forms that are flexible to business domain, use experience, and channel.
+
+## Form layout
+
+Form controls align to a column layout with an even number of columns across breakpoints.
+
+<ImageSwitcher
+  images={[
+    { src: "/img/patterns/forms/form_layout_1.png", alt: "" },
+    { src: "/img/patterns/forms/form_layout_2.png", alt: "" },
+  ]}
+  label="Show spacing"
+/>
+
+\*You can find more information on the Salt spacing system here.
```

Comment:
is this supposed to be a link? :)

Reply:
I still need to go through and add the links everywhere :)